src/book/01-team/01-introduction/03-current-team.mdx

```diff
@@ -14,75 +14,84 @@ slug: /team/current-members
 
-- **Cameron Murtagh**: Undergraduate student studying a combined degree in Mathematics and Computer Science.
-
-- **Joel Wong**: Currently studying for a Doctorate of Philosophy in Computer Science. Specialises in computer vision.
+### Junior Members
 
-### New Members
-
-These members joined NUbots in our latest recruitment drive.
+This is the starting level of membership. Generally, these members have been on the team for less than a year and are working on their first project.
 
 - Shaan Arora, Computer Science
+- Eleanor Bartley, Electrical Engineering
+- Peta Carlyle, Software Engineering
+- Thisara Chanthuranga, Electrical Engineering
+- Lachlan Court, Computer Science
 - Liam Craft, Computer Science
-- Lachlan Drake, Electrical Engineering
+- Yosiah de Koeyer, Computer Engineering and Computer Science
+- Lucy Dowdell, Mathematics (Advanced)
 - Ryan Duffield, Mechatronics Engineering
-- Nathan Gervasoni, Computer Science
-- Robin Griffiths, Electrical Engineering/Computer Systems Engineering
-- Abigail Hall, Mathematics/Computer Science
+- Nicholas Dziura, Computer Engineering
+- Abigail Hall, Mathematics and Computer Science
 - Yuji Ishikawa, Computer Science
 - Damon Kinkade, Computer Science
+- Sam McFarlane, Mechatronics Engineering and Electrical Engineering
+- Nicholas Milford, Mechatronics Engineering
+- Alana Noonan, Computer Science
 - Thomas O'Brien, Mechatronics Engineering
-- Mikyla Peters, Electrical Engineering/Computer Engineering
-- Aden Weinmann, Mechatronics Engineering/Physics
+- Mikyla Peters, Electrical Engineering and Computer Engineering
+- Peter Salem, Medical Engineering
+- Cameron Steggles, Mathematics
+- Bryce Tuppurainen, Software Engineering
 - Benjamin Young, Computer Systems Engineering
 
 ## Supporting Members
 
 Past members of the team that are still involved with projects, mentoring, and support.
 
+- **Matt Amos**: Computer Science and Computer Engineering Honours graduate. Specialises in synthetic dataset generation and robotic movement.
+
 - **Brendan Annable**: Bachelor of Software Engineering Graduate. Ex-Team Leader. Lead Developer of NUsight.
 
-- **Dr. Trent Houliston**: Holds a Doctorate of Philosophy in Software Engineering. Ex-Team Leader. NUbots mentor. Currently working at 4Tel. Specialises in Software Architecture, Machine Learning and Computer Vision.
+- **Alex Biddulph**: Bachelor of Computer Science, Bachelor of Computer Engineering with Honours. Currently studying for a Doctorate of Philosophy in Computer Engineering. Ex-Team Leader. Specialises in computer vision.
 
-- **Matt Amos**: Computer Science and Computer Engineering Honours graduate. Specialises in synthetic dataset generation and robotic movement.
+- **Dr. Trent Houliston**: Holds a Doctorate of Philosophy in Software Engineering. Ex-Team Leader. NUbots mentor. Currently working at 4Tel. Specialises in Software Architecture, Machine Learning and Computer Vision.
```

Comment:
Alex and Trent specialise? I thought they knew everything ;P

Reply:
:laughing: you're not wrong